llvm/lib/Target/AMDGPU/AMDGPUSplitModule.cpp

```diff
@@ -562,6 +564,9 @@ void SplitGraph::buildGraph(CallGraph &CG) {
 
         if (handleCalleesMD(Inst, KnownCallees))
           continue;
+        // If we failed to parse any !callees MD, or some was missing,
+        // the entire KnownCallees list is now unreliable.
```

Comment:
Sounds like the verifier is missing validation on callees entries? You shouldn't really have to worry about parsing it